tools/code-analyzer/src/commands/analyzer/index.ts

```diff
@@ -445,11 +448,12 @@ export default class Analyzer extends Command {
 
 				const name = getHookName( hookName[ 3 ] );
 
-				const description = getHookDescription( raw, name );
+				const description = getHookDescription( raw, name ) || '';
```

Comment:
This was upsetting TS since `getHookDescription` returns `string | false`, we coerce to empty string if its false here.